source/guides/zepto-api.md

```diff
@@ -522,10 +522,10 @@ Failed transactions will contain the following information inside the event:
 
 ### Using failure codes
 
-> - [DE credit failure codes]("#de-credit-failures")
-> - [DE debit failure codes]("#de-debit-failures)
+> - [DE credit failure codes](https://help.zepto.money/en/articles/5633407-au-transaction-failure-responses#h_b2229b3745)
+> - [DE debit failure codes](https://help.zepto.money/en/articles/5633407-au-transaction-failure-responses#h_b15d7bd439)
 
-To simulate a transaction failure, create a Payment or Payment Request with an amount corresponding to the desired [failure code](#failure-codes).
```

Comment:
These removed references link to pages which are not available in the readme-hosted API docs: https://docs.zeptopayments.com/docs/.

Not sure exactly what's going on there but I will investigate separately. For now, I'll change these links to help centre articles which do include the error codes.